src/components/SocialHeader.module.css

```diff
@@ -0,0 +1,13 @@
+.socialHeader {
+    background-color: white;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    border-bottom: 1px solid rgba(15,70,100,.12);
+}
+
+.socialHeader h1 {
+    margin: 0;
+    padding: 0.5rem;
+    font-size: 2rem;
+    text-align: center;
+}
```

Comment:
Add a [closing line break](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) to EOF please.

Reply:
Done, thanks!